Source/Libraries/GSF.Core/Collections/CollectionExtensions.cs

```diff
@@ -422,27 +422,69 @@ public static bool AllParallel<TSource>(this IEnumerable<TSource> source, Func<T
         /// <typeparam name="T"><see cref="Type"/> of elements in the <paramref name="source"/></typeparam>
         /// <param name="source">An enumeration over which to find the median element(s).</param>
         /// <returns>The median item(s) from an enumeration, or <c>null</c> if <paramref name="source"/> is <c>null</c>.</returns>
-        public static T[] Median<T>(this IEnumerable<T> source)
+        public static IEnumerable<T> Median<T>(this IEnumerable<T> source)
         {
             if (source == null)
```

Comment:
Better as `is null`, although the result of the functions in the files still use `== null`:
```c#
if (source is null)
```